Lib/test/test_pdb.py

```diff
@@ -3235,6 +3237,32 @@ def test_checkline_is_not_executable(self):
                 self.assertFalse(db.checkline(os_helper.TESTFN, lineno))
 
 
+@support.requires_subprocess()
+class PdbTestReadline(unittest.TestCase):
+    def setUpClass():
+        # Ensure that the readline module is loaded
+        # If this fails, the test is skipped because SkipTest will be raised
+        readline = import_module('readline')
+        if readline.__doc__ and "libedit" in readline.__doc__:
+            raise unittest.SkipTest("libedit readline is not supported for pdb")
+
+    def test_basic_completion(self):
+        script = textwrap.dedent("""
+            import pdb; pdb.Pdb().set_trace()
+            print('hello')
```

Comment:
(see the other suggestion)
```suggestion
            # Concatenate strings so that the output doesn't appear in the source
            print('hello' + '!')
```